<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(95, 151, 235, 255), stop:1 rgba(255, 255, 255, 255));

QPushButton-active {
	position: relative;
	top: 1px;
}

QPushButton{
	background-color: rgb(0, 255, 0);
	color: rgb(255, 255, 255);
	border-radius: 8px;
	border-style: outset;
}

QPushButton:hover {
	color: light-grey;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(95, 151, 235, 255), stop:1 rgba(255, 255, 255, 255));

QPushButton-active {
	position: relative;
	top: 1px;
}

QPushButton{
	background-color: rgb(0, 255, 0);
	color: rgb(255, 255, 255);
	border-radius: 8px;
	border-style: outset;
}

QPushButton:hover {
	color: light-grey;
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>399</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>338</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="FilterButton">
        <property name="font">
         <font>
          <family>Arial</family>
         </font>
        </property>
        <property name="text">
         <string>Filter</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="2">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="AddBranchButton">
          <property name="minimumSize">
           <size>
            <width>81</width>
            <height>21</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton-active {
	position: relative;
	top: 1px;
}

QPushButton{
	background-color: rgb(0, 255, 0);
	color: rgb(255, 255, 255);
	border-radius: 8px;
	border-style: outset;
}

QPushButton:hover {
	color: light-grey;
}</string>
          </property>
          <property name="text">
           <string>Add Branch</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="EditBranchButton">
          <property name="minimumSize">
           <size>
            <width>81</width>
            <height>21</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton-active {
	position: relative;
	top: 1px;
}

QPushButton{
	background-color: rgb(0, 255, 0);
	color: rgb(255, 255, 255);
	border-radius: 8px;
	border-style: outset;
}

QPushButton:hover {
	color: light-grey;
}</string>
          </property>
          <property name="text">
           <string>Edit Branch</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="DeleteBranchButton">
          <property name="minimumSize">
           <size>
            <width>81</width>
            <height>21</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton-active {
	position: relative;
	top: 1px;
}

QPushButton{
	background-color: rgb(0, 255, 0);
	color: rgb(255, 255, 255);
	border-radius: 8px;
	border-style: outset;
}

QPushButton:hover {
	color: light-grey;
}</string>
          </property>
          <property name="text">
           <string>Delete Branch</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2" rowspan="2">
       <layout class="QVBoxLayout" name="RelationshipVertLayout">
        <item>
         <widget class="QLabel" name="RelationshipLabel">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1" rowspan="2">
       <widget class="QWidget" name="ChartWidget" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: transparent;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
